docs/source/entities-advanced.mdx

```diff
@@ -281,53 +281,73 @@ type Bill @key(fields: "id") {
 
 </CodeColumns>
 
-The [`@override` directive](./federated-types/federated-directives#override) enables us to perform this migration incrementally with no downtime.
+The [`@override` directive](./federated-types/federated-directives#override) enables you to incrementally migrate between subgraphs with no downtime.
 
-First, we deploy a new version of the Billing subgraph that defines _and_ resolves the `Bill` fields we want to move:
+Follow these steps to use the `@override` directive:
 
-<CodeColumns>
+1. If the `@override` directive isn't already imported, include it in your schema's `@link` imports:
 
-  ```graphql title="Payments subgraph"
-  type Bill @key(fields: "id") {
-    id: ID!
-    amount: Int!
-    payment: Payment
-  }
+    ```graphql {3} title="Billing subgraph"
+    extend schema
+      @link(url: "https://specs.apollo.dev/federation/v2.3",
+            import: ["@key", "@shareable", "@override"])
+    ```
 
-  type Payment {
-    # ...
-  }
-  ```
+2. Deploy a new version of the Billing subgraph that both defines and resolves the `Bill` fields you want to move:
 
-  ```graphql {3} title="Billing subgraph"
-  type Bill @key(fields: "id") {
-    id: ID!
-    amount: Int! @override(from: "Payments")
-  }
-  ```
+    <CodeColumns>
 
-</CodeColumns>
+      ```graphql title="Payments subgraph"
+      type Bill @key(fields: "id") {
+        id: ID!
+        amount: Int!
+        payment: Payment
+      }
+
+      type Payment {
+        # ...
+      }
+      ```
 
-The `@override` directive says, "Resolve this field in this subgraph _instead of_ in the Payments subgraph."
+      ```graphql {3} title="Billing subgraph"
+      type Bill @key(fields: "id") {
+        id: ID!
+        amount: Int! @override(from: "Payments") #highlight-line
+      }
+      ```
 
+    </CodeColumns>
 
-<blockquote>
+      Applying the `@override` directive tells the router to resolve the `amount` field in the Billing subgraph instead of the Payments subgraph.
 
-In any subgraph where you use `@override`, make sure to include it in your schema's `@link` imports (code-first subgraph libraries usually do this for you):
 
-```graphql {3} title="Billing subgraph"
-extend schema
-  @link(url: "https://specs.apollo.dev/federation/v2.3",
-        import: ["@key", "@shareable", "@override"])
-```
+3. Update your router's supergraph schema to migrate to the updated Billing subgraph, either gradually or all at once.
 
-</blockquote>
+    * **To migrate gradually**, use the [progressive `@override` feature](/router/configuration):
+
+      1. Configure your router to enable progressive `@override`:
```

Comment:
This step not needed, instead describe that router auto-detects usages of progressive override (has `labels`)